packages/wiz/changelog.yml

```
@@ -1,26 +1,21 @@
 # newer versions go on top
-- version: "1.9.0-preview05"
+- version: "2.0.0"
   changes:
+    - description: Remove rule.references field and adjust rule.* mappings.
+      type: enhancement
```

> and adjust rule.* mappings.

When I read this I'm wonder what adjustments were made.

I also think the changelog should tell user how to deal with the breaking change if they were using the removed field.

```suggestion
    - description: Remove rule.references field and its mapping. Please use the ECS rule.reference field instead.
      type: breaking-change
```

---

thanks, makes sense!